_posts/2020-03-16-dask-usage-at-blue-yonder.markdown

```diff
@@ -0,0 +1,159 @@
+---
+layout: single
+title:  "Dask Usage at Blue Yonder"
+date:   2020-03-16 10:00:00 +0100
+tags: technology python data-engineering
+header:
+  overlay_image: assets/images/tech_gear_banner.jpg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Andreas Merkel, Kshitij Mathur
```

Comment:
You need to have the entry of `Andreas Merkel, Kshitij Mathur` in `authors.yml` for this to "work", e.g. https://github.com/JDASoftwareGroup/jdatech.github.io/blob/1b4ab751c9946321272f4a918fd81db509113af7/_data/authors.yml#L71
(You can test this locally using the last command provided in the readme)

Reply:
Thx, I was already suspecting it's not that smart...